modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSkusVirtualSettingsOverride.testcase

```diff
@@ -37,6 +37,152 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "COMMERCE-11379 | To verify that a buyer user can download the virtual product by the URL retrieved using API when SKU override is enabled"
+	@priority = 4
+	test CanDownloadConfiguredVirtualProductWithSKUOverride {
+		property portal.acceptance = "false";
+
+		var baseURL = PropsUtil.get("portal.url");
+
+		task ("Given a virtual product with Activation Status is set") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = '''"Virtual T-Shirt''');
+
+			CommerceNavigator.gotoEntry(entryName = "Virtual T-Shirt");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Virtual");
+
+			CommerceProducts.configureVirtualProduct(
+				activationStatus = "Processing",
+				assertUploaded = "true",
+				attachment = "Commerce_Black.jpg",
+				maxDownloads = 2);
+		}
+
+		task ("And the admin enables the override for the SKU") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = '''"Virtual T-Shirt"''');
+
+			CommerceNavigator.gotoEntry(entryName = "Virtual T-Shirt");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceNavigator.gotoTableListEntry(entryName = "VIRTUALSKU");
+
+			CommerceProducts.updateProductVirtualSettingOverride(
+				activationStatus = "Pending",
+				key_contentType = "Base Information",
+				key_fieldLabel = "Activation Status",
+				maxDownloads = 3,
+				virtualProductFile = "Commerce_Black.jpg");
```

Comment:
@Brunno0liveira to make sure that the override is actually changing the file, can you change the virtualProductFile to "Commerce_White.jpg"? Thanks